tutorials/discussions/plot_background_statistics.py

```diff
@@ -46,7 +46,7 @@
 # some probability (e.g., p < 0.05). This probability is also called the
 # significance level :math:`\alpha`.
 # To think about what this means, let's follow the illustrative example from
-# [1]_ and construct a toy dataset consisting of a 40 x 40 square with a
+# :footcite:`RidgwayEtAl2012` and construct a toy dataset consisting of a 40 x 40 square with a
```

Comment:
this will yield a "line too long" error from our style checker.  Max line length is 79 characters.

Reply:
Thanks for the heads up